src/lib/api/archive.repo.spec.ts

```diff
@@ -0,0 +1,73 @@
+import anyTest, { TestInterface } from 'ava';
+import axios from 'axios';
+import * as sinon from 'sinon';
+
+import { PermanentApiRequestData } from '../model';
+
+import { ArchiveRepo } from './archive.repo';
+import { CsrfStore } from './csrf';
+
+const test = anyTest as TestInterface<{
+  archiveRepo: ArchiveRepo;
+  csrfStore: CsrfStore;
+}>;
+
+const apiKey = 'apiKey';
```

Comment:
We probably don't need a variable for this since it is only used in one place (also lets make it something that clearly shouldn't change, like `'fakeApiKeyForTests'` lest someone think that it needs to be populated with a real API key)